test/extended/prometheus/prometheus.go

```diff
@@ -335,12 +335,12 @@ func (t *prometheusTargets) Expect(l labels, health, scrapeURLPattern string) er
 
 type labels map[string]string
 
-func (l labels) With(name, value string) labels {
+func (l labels) With(name, Value string) labels {
```

Comment:
why is this needed?

Reply:
I believe this still needs addressing, I don't see a need for this change.